encodings/fsst/benches/fsst_compress.rs

```diff
@@ -65,6 +67,26 @@ fn train_compressor(bencher: Bencher, args: (usize, usize, u8)) {
     bencher.bench_local(move || fsst_train_compressor(&array).unwrap())
 }
 
+#[divan::bench(args = BENCH_ARGS)]
+fn pushdown_compare(bencher: Bencher, args: (usize, usize, u8)) {
+    let (string_count, avg_len, unique_chars) = args;
+    let array = generate_test_data(string_count, avg_len, unique_chars);
```

Comment:
I might be confusing something here, but generate_test_data returns a VarBinArray, not an FSST array?

Reply:
@gatesn Indeed, will change this to FSST array!